chains/ethereum/online_chain_driver.go

```diff
@@ -136,7 +136,7 @@ func (acc *ETHChainDriver) SignRedeem(fromaddr common.Address, redeemAmount *big
 	if err != nil {
 		return nil, err
 	}
-	gasLimit := uint64(6721974)
+	gasLimit := uint64(700000)
```

Comment:
Can we add this as a constant somewhere.. so that we dont need to change it at multiple places?

Reply:
Constant?